guides/common/modules/proc_listing-using-satellite-ansible-modules.adoc

```diff
@@ -18,39 +17,28 @@ At the time of writing, the `ansible-doc -l` command does not list collections y
 endif::[]
 
 ifeval::["{build}" != "satellite"]
+
 Starting with Ansible 2.10, you can view the installed {Project} Ansible modules by running:
 
----
-# ansible-doc -l theforeman.foreman
----
+[options="nowrap" subs="+quotes,attributes"]
+----
+# ansible-doc -l {ansible-namespace}
+----
 
 When using Ansible before 2.10, you can view the installed {Project} Ansible modules by listing the content of the following directory:
 
+[options="nowrap" subs="+quotes,attributes"]
 ----
-# ls /usr/share/ansible/collections/ansible_collections/theforeman/foreman/plugins/modules/
+# ls {ansiblefilepath}
 ----
 
-Alternatively, you can also see the complete list of {Project} Ansible modules and other related information at https://galaxy.ansible.com/theforeman/foreman in the Ansible Galaxy.
-
 endif::[]
 
+Alternatively, you can also see the complete list of {Project} Ansible modules and other related information at [{ansible-galaxy}]({ansible-galaxy}).
```

Comment:
This does not build right. You can only keep a URL - the attribute - without formatting.